identity_iota/src/did/did.rs

```diff
@@ -17,8 +17,8 @@ pub fn method_id(did: &DID) -> Result<&str> {
 
 /// Creates an 81 Trytes IOTA address from the DID
 pub fn create_address_hash(did: &DID) -> Result<String> {
-    let digest: &[u8] = &Blake2b256::digest(method_id(did)?.as_bytes());
-    let encoded: String = encode(digest).into_string();
+    let digest: MultihashGeneric<_> = Blake2b256::digest(method_id(did)?.as_bytes());
```

Comment:
I think we should remove this hashing part here and use the method_id right away